sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/metadata.yaml

```diff
@@ -23,7 +23,7 @@ scheduling:
     execution_delta: 1h
   - task_id: search_derived__mobile_search_clients_daily__v1
     dag_name: bqetl_mobile_search
-    execution_delta: 1h
+    execution_delta: 0h
```

Comment:
Is there anything this gets set to by default? According to Airfow docs if `execution_delta` is not provided it default to checking the upstream task with the same execution_date:

```
execution_delta (Optional[[datetime.timedelta](https://docs.python.org/3/library/datetime.html#datetime.timedelta)]) – time difference with the previous execution to look at, the default is the same logical date as the current task or DAG. For yesterday, use [positive!] datetime.timedelta(days=1). Either execution_delta or execution_date_fn can be passed to ExternalTaskSensor, but not both.
```

https://airflow.apache.org/docs/apache-airflow/2.3.3/_api/airflow/sensors/external_task/index.html?highlight=external_task#module-airflow.sensors.external_task